src/Polytopes/LinearProgram.jl

```diff
@@ -82,7 +96,34 @@ feasible_region(lp::LinearProgram) = lp.feasible_region
 ###############################################################################
 ###############################################################################
 
-
+"""
+    minimal_vertex(LP::LinearProgram)
+
+Return, if it exists, a point of the feasible region of `LP` which minimizes the objective
+function of `LP`.
+
+Return `nothing` if the objective function does not obtain a minimal value over the feasible region.
```

Comment:
```suggestion
Return either a point of the feasible region of `LP` which minimizes the objective
function of `LP`, or `nothing` if no such point exists.
```
If you prefer to keep your current text, please change "obtain" to "attain".